bridges/RedditBridge.php

```diff
@@ -0,0 +1,59 @@
+<?php
+class RedditBridge extends FeedExpander {
+
+	const MAINTAINER = 'leomaradan';
+	const NAME = 'Reddit Bridge';
+	const URI = 'https://www.reddit.com/';
+	const DESCRIPTION = 'Reddit bridge for Feedly';
```

Comment:
You could replace this by:

> Reddit RSS Feed fixer

Or something like this, this is independent of Feedly.